uint8 HAND_STATE_UNKNOWN = 0
uint8 HAND_STATE_NOT_TRACKED = 1
uint8 HAND_STATE_OPEN = 2
uint8 HAND_STATE_CLOSED = 3
uint8 HAND_STATE_LASSO = 4

kinect_bridge2/KinectJoint[] joints
uint8 is_tracked
uint8 hand_state_left
uint8 hand_state_right
